libs/elements/layout/src/lib/pages/elewa-group-main-page/elewa-group-main-page.component.ts

```diff
@@ -1,8 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'elewa-group-elewa-group-main-page',
   templateUrl: './elewa-group-main-page.component.html',
   styleUrls: ['./elewa-group-main-page.component.scss'],
 })
-export class ElewaGroupMainPageComponent {}
+export class ElewaGroupMainPageComponent implements OnInit {
+  route: string;
+  constructor(public router: Router) {}
+  ngOnInit(): void {
+    this.route = this.router.url;
```

Comment:
This will return a full url and url will defer depending on language but is still same page i.e. home/en, home/fr and home/nl

Reply:
The url needs to be split and fetch page by index